core/system_metric/sys_metric_stat.go

```diff
@@ -178,7 +180,11 @@ func retrieveAndUpdateCpuStat() {
 
 	cpuRatioGauge.Set(cpuPercent)
 
-	currentCpuUsage.Store(cpuPercent)
+	numcpu := runtime.NumCPU()
+	cpuUsage := cpuPercent / float64(numcpu) / float64(100)
+	cpuUsage = math.Max(0.0, cpuUsage)
```

Comment:
When can it be less than 0.0 or greater than 1.0

Reply:
I find process.Percent in gopsutil library may return very large invalid value when have mary goroutines, this will cause cpuUsage > 1. So I add code "cpuUsage = math.Min(1.0, cpuUsage)". 
I submit issure in gopsutil to report bug, https://github.com/shirou/gopsutil/issues/1214.

process.Percent can not return value <0, the code "cpuUsage = math.Max(0.0, cpuUsage)" is useless and redundant.

Reply:
cpuUsage = math.Max(0.0, cpuUsage) is not required， can be removed